bounded-core/src/main/scala/io/cafienne/bounded/akka/persistence/eventmaterializers/offsetstores/CassandraOffsetStore.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2016-2018 Cafienne B.V. <https://www.cafienne.io/bounded>
+ */
+
+package io.cafienne.bounded.akka.persistence.eventmaterializers.offsetstores
+
+import java.util.UUID
+
+import akka.persistence.cassandra.query.scaladsl.CassandraReadJournal
+import akka.persistence.query.{Offset, Sequence, TimeBasedUUID}
+import io.cafienne.bounded.akka.persistence.eventmaterializers.{EventMaterializerExecutionContext}
+
+import scala.concurrent.duration._
+import scala.concurrent.{Await, Future}
+
+class CassandraOffsetStore(readJournal: CassandraReadJournal, createTableTimeout: Duration) extends OffsetStore {
+
+  import EventMaterializerExecutionContext._
+
+  Await.result(
+    readJournal.session.executeWrite(
+      "CREATE TABLE IF NOT EXISTS akka.vw_offsetstore (view_identifier text PRIMARY KEY, offset_type text, offset_value text);"
```

Comment:
It would be nice if storagespace name (here: akka) could be configurable, to support sharing cassandra servers over multiple applications.

Reply:
I believe that the space used by akka is configured by akka. We could use that.